ddtrace/ddtrace.go

```diff
@@ -114,7 +114,7 @@ func NewStartSpanConfig(opts ...StartSpanOption) StartSpanConfig {
 // Logger implementations are able to log given messages that the tracer or profiler might output.
 type Logger interface {
 	// Log prints the given message.
-	Log(msg string)
+	Log(lvl log.Level, msg string)
```

Comment:
⚠️ I think there is a problem here. `log.Level` is defined in the `internal/log` package, which is not importable outside of dd-trace-go. So as far as I can tell, this would make it impossible for our users to define their own loggers.

So if we want to make this change, we need to make the `log.Level` type public, e.g. by exporting it as `ddtrace.LogLevel`.